charts/tidb-cluster/templates/config/_prometheus-config.tpl

```diff
@@ -46,11 +53,63 @@ scrape_configs:
     - source_labels: [__meta_kubernetes_pod_ip]
       action: replace
       target_label: kubernetes_pod_ip
+    {{- if .Values.enableTLSCluster }}
```

Comment:
Add a TODO here that when TiKV supports metrics API over https, this workaround should be removed.